release.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python2.7
+
+import semver
+import xml.etree.ElementTree as ET
+import sys 
+
+"""Type: Major/Minor/Patch"""
+
+"""
+1. Read in values.xml
+2. Increment the SDK version value based on correct semVers
+3. Return: new version number
+"""
+def get_current_version(root):
+  for sdk_version in root.iter("sdk_version"):
+    return sdk_version.text
+
+def update_version(root, xml, version):
+  for sdk_version in root.iter("sdk_version"):
+    sdk_version.text = str(version)
+    sdk_version.set('updated', 'yes')
+    xml.write("AndroidSDK/res/values/strings.xml")
+
+def main():
+  type = sys.argv[1]
```

Comment:
`type` is a reserved word in python. It lets you overwrite it but it's not good practice. You can make this more descriptive like `release_type`. https://docs.python.org/3/library/functions.html#type

Reply:
done